.github/workflows/smoke-test.yml

```diff
@@ -97,6 +93,14 @@ jobs:
         run: |
           brew install qemu socket_vmnet
           HOMEBREW=$(which brew) && sudo ${HOMEBREW} services start socket_vmnet
+      - name: Clean Mac (macos)
+        if: contains(matrix.os, 'macos')
+        run: |
+          sudo killall socketfilterfw bootpd
+          sudo mdutil -a -i off
+          sudo pkill -f "mds_stores|mds|mdworker_shared|mdworker|spotlightknowledged|Spotlight|photoanalysisd|cloudphotod|mediaanalysisd|analyticsd|mediaanalysisd|mdwrite|corespotlightd|geod|weatherd"
```

Comment:
Please don't, I don't do any of these in vment-helper and they have no issues.

Reply:
vmnnet helper process doesnt need as much as minikube